bugsnag-android-core/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -82,6 +82,9 @@
     private PluginClient pluginClient;
 
     final Notifier notifier = new Notifier();
+    LastRunInfo lastRunInfo;
```

Comment:
This jumps out as having no modifier – any reason it can't be private and final?

Reply:
This would generate a [synthetic accessor](https://medium.com/thoughts-overflow/effects-of-javas-synthetic-accessor-methods-in-android-bb67b3bac22e) as it's accessed from an inner class, which increases the library size unnecessarily. Our static analysis would fail if this were declared as private. The loading also needs to be asynchronous to avoid unnecessary work on the main thread, so it's not possible to declare it as final.